### PR TITLE
transcriber: Do not infinite loop if the current parsed node is an error

### PR DESCRIPTION
**Pull request by CohenArthur**

### CODE REVIEW

File: gcc/rust/expand/rust-macro-expand.cc [hunk 1]
```diff
@@ -727,6 +727,14 @@ parse_many (Parser<MacroInvocLexer> &parser, TokenId &delimiter,
 	break;
 
       auto node = parse_fn ();
+      if (node.is_error ())
+	{
+	  for (auto err : parser.get_errors ())
+	    err.emit_error ();
+
+	  break;
```
Comment (philberty) [inline gcc/rust/expand/rust-macro-expand.cc:735]: what about a return instead of a break?

**Review by philberty — APPROVED** [review]
LGTM

File: gcc/rust/expand/rust-macro-expand.cc (hunk 1 above)
Comment (CohenArthur) [reply gcc/rust/expand/rust-macro-expand.cc]: Fixed in https://github.com/Rust-GCC/gccrs/commit/8dfa9676db0f8660c6c745adb59ef6323dca2341 :)

**Pull request merged**